src/components/Sum.jsx

```diff
@@ -1 +1,17 @@
 // export function Sum() {}
```

Comment:
don't forget to delete redundant comments

Reply:
delete it